### DISCUSSION TITLE
Add support for countDocuments

### DISCUSSION THREAD

**Discussion by mhdzumair** [BeanieODM/beanie#926] [Feature Request]
Hi, i see that the model doesn't have function for countDocuments as in the mongo. According to mongo doc, https://www.mongodb.com/docs/manual/reference/method/db.collection.count/#inaccurate-counts-without-query-predicate

I haven't seen any inaccurate results, however I felt like a bit of difference in response time when using the below code. I'm not 100% about the benchmark due to using external DB through internet.

```
count = await TorrentStreams.count()
```

```
collection = TorrentStreams.get_motor_collection()
count = await collection.count_documents({})
```

**Comment by staticxterm** [comment]
Hi,

Beanie's `Document.count()` method (part of the FindQuery class: [beanie/odm/queries/find.py](https://github.com/BeanieODM/beanie/blob/10749070ac20ca2183ee98f53a44c1ea58d9d17a/beanie/odm/queries/find.py#L157)) under the hood already uses PyMongo's `count_documents()` method, and does not suffer from the mentioned deficiencies.
For reference, here is PyMongo's documentation that is the only one relevant and valid here: https://pymongo.readthedocs.io/en/4.15.3/api/pymongo/collection.html#pymongo.collection.Collection.count_documents
